### PR TITLE
SSB4: Adjust aegii's set

### PR DESCRIPTION
**Pull request by Volco**
idk if I'm missing something, might be because I was feeling a bit tired when I did this and I suck at reviewing my own stuff for missing parts when tired.

### CODE REVIEW

**Comment by Volco** [comment]
i forgot to commit the travis fix before i pr'd lol

**Comment by ghost** [comment]
I genuinely hate how this is implemented but also I can't find a way to make this any less hacky so GG WP. My concern is that this will crash if Morfent or trace get a hold of this ability it will (probably) crash, so make a check for that.

**Comment by Volco** [comment]
i hate it too lol 

okay going to add the check then test it, i'll @ you when it's added

**Comment by Volco** [comment]
the only other thing i can think of is a "skip" forme like some of the other mons have but that seems a bit much for only the moveset changing

**Comment by Volco** [comment]
@xInstruct

**Comment by ghost** [comment]
👍

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -127,6 +127,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		},
 		onHit(pokemon) {
 			pokemon.addVolatile('stall');
+			pokemon.m.swapSets();
```
Comment (ghost) [inline data/mods/ssb/moves.ts:131]:
```suggestion
			if (pokemon.species.baseSpecies !== 'Aegislash') return;
			pokemon.m.swapSets();
```

Comment (ghost) [reply data/mods/ssb/moves.ts:131]: blame grimAuxiliatrix and/or boat for this

Comment (Volco) [reply data/mods/ssb/moves.ts:131]: oh right i forgot transform exists too lol

**Pull request merged**